sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/instrumentation/otel/OTelInstrumentation.java

```diff
@@ -92,15 +106,16 @@ public OTelInstrumentation(InstrumentationOptions applicationOptions,
         this.otelInstance = explicitOTel;
         this.libraryOptions = libraryOptions;
         this.isTracingEnabled = applicationOptions == null || applicationOptions.isTracingEnabled();
+        this.isMetricsEnabled = applicationOptions == null || applicationOptions.isMetricsEnabled();
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public Tracer getTracer() {
+    public Tracer createTracer() {
         if (isTracingEnabled && OTelInitializer.isInitialized()) {
-            Object otelTracerProvider = GET_PROVIDER_INVOKER.invoke(getOtelInstance());
+            Object otelTracerProvider = GET_TRACER_PROVIDER_INVOKER.invoke(getOtelInstance());
```

Comment:
nit: Do we also want to rename the invoker to `CREATE_...`

Reply:
invokers are called based on the method they represent. OTel has a cache of tracers/meters/etc, so for them it's `get`. We don't have a cache and we don't provide convenience API for the end users, so I want to use `create` to make everyone aware.